geode-core/src/main/java/org/apache/geode/distributed/internal/locks/DLockGrantor.java

```diff
@@ -146,6 +148,10 @@
    */
   private final TXReservationMgr resMgr = new TXReservationMgr(false);
 
+  private final Map<InternalDistributedMember, Long> membersDepartedTime = new HashMap();
+  private final int membersDepartedTimeKeptSize = 10;
+  private final long departedMemberKeptInMapMilliSeconds = 24 * 60 * 60 * 1000;
```

Comment:
Use that TimeUnit jdk calls to calculate this. I think you could say something like DAY.toMillis().